Makefile

```diff
@@ -17,7 +17,7 @@ dump.sql.gz:
 .PHONY: services
 services:
 	docker-compose up -d
-	@echo Info: To (re-)init DB run "make initdb" now.
+	@echo Info: To \(re-\)init DB run "make initdb" now.
```

Comment:
argh, here you see my lack of Makefile editing experience :+1:

Reply:
It's not something I knew either :stuck_out_tongue: